src/main.cpp

```diff
@@ -48,6 +49,9 @@ void initialize_overlap_parser(seqan3::argument_parser & parser, OverlapOptions
     parser.add_option(options.input_path, 'i', "input_bam",
                       "The name of the SAM/BAM file to query.", seqan3::option_spec::required,
                       seqan3::input_file_validator{{"sam", "bam"}});
+    parser.add_option(options.out_file, 'o', "output_sam",
+                      "The SAM file, where the results should be stored.", seqan3::option_spec::required,
+                      seqan3::input_file_validator{{"sam"}});
```

Comment:
```suggestion
    parser.add_option(options.out_file, 'o', "output_sam",
                      "The SAM file, where the results should be stored.", seqan3::option_spec::standard,
                      seqan3::input_file_validator{{"sam"}});
```

This way, they can provide an -o filename. Otherwise, it will output to std::cout (important because people will often want the results outputted to std::cout to directly pipe them to another tool.)